homeassistant/components/vicare/__init__.py

```diff
@@ -113,8 +118,17 @@ def get_supported_devices(
     devices: list[PyViCareDeviceConfig],
 ) -> list[PyViCareDeviceConfig]:
     """Remove unsupported devices from the list."""
+
     return [
         device_config
         for device_config in devices
         if device_config.getModel() not in UNSUPPORTED_DEVICES
     ]
+
+
+def get_online_devices(
```

Comment:
Instead of ignoring offline devices, all entities of offline devices should be marked as unavailable